substrate/client/consensus/manual-seal/src/lib.rs

```diff
@@ -467,7 +467,9 @@ mod tests {
 		assert_eq!(client.header(created_block.hash).unwrap().unwrap().number, 1)
 	}
 
-	#[tokio::test]
+	// TODO: enable once the flakiness is fixed
```

Comment:
Got a tracking issue to link to?

Reply:
https://github.com/paritytech/polkadot-sdk/issues/3603